### ISSUE TITLE
Obsidian 0.15+ Bug: tree.iterate has changed syntax

### ISSUE THREAD

**Issue by SlRvb** [mdelobelle/obsidian_supercharged_links#126]
I think I managed to narrow down which plugin was causing some wild console log behavior and it seems to have come from supercharged links. When I toggle supercharged links off, I don't get these message anymore.

Mostly getting a _bunch_ of `tree.iterate has changed syntax` errors when my cursor hovers over a link. 🥴 

![FbfhC6Hrh8](https://user-images.githubusercontent.com/54087190/174648216-caab4ccb-1b23-453c-885a-fd8193a8a75f.png)

**Comment by HEmile** [comment]
Yeah, I saw these too. Will be my first priority to fix once I get back into plugin dev...

**Comment by kometenstaub** [comment]
Just a pointer, the fix that needs to be done looks very similar to here:
https://github.com/artisticat1/obsidian-latex-suite/commit/5dea474d7f4dc05bf2d4866888b6110e38bb4b5c

**Comment by nothingislost** [comment]
PR for the fix here https://github.com/mdelobelle/obsidian_supercharged_links/pull/128
